core/templates/dev/head/components/skill-mastery/skill-mastery.directive.ts

```diff
@@ -46,8 +49,28 @@ angular.module('oppia').directive('skillMasteryViewer', [
             return Math.round(ctrl.skillMasteryDegree * 100);
           };
 
+          ctrl.getMasteryChangePercentage = function() {
+            if (ctrl.masteryChange >= 0) {
+              return '+' + Math.round(ctrl.masteryChange * 100);
+            } else {
+              return Math.round(ctrl.masteryChange * 100);
+            }
+          };
+
           ctrl.getLearningTips = function() {
-            // TODO(sophiewu6): Implement learning tips feature.
+            if (ctrl.masteryChange > 0) {
+              if (ctrl.skillMasteryDegree >= GOOD_MASTERY_CUTOFF) {
+                return 'You have mastered this skill very well! ' +
+                  'You can work on other skills or learn new skills.';
+              } else {
+                return 'You have made progress! You can increase your ' +
+                  'mastery level by doing more practice sessions.';
+              }
+            } else {
```

Comment:
I think we want to add a link to the word 'concept card' and have it open a concept card modal

Reply:
It there any specific reason why link is better? I am adding a button because in this way learners can view the concept card even when the learning tip does not contain "concept card" (when mastery increased).

Reply:
The reason I mentioned a link is because it may not be obvious to the user that the concept card we speak about in the text can be opened by the bottom of the modal. If we keep it as a link however, the user will know that they can open the concept card from that link

Reply:
I added a screenshot above. Personally, I think the button is more straightforward since it's highlighted in color green, and even when the learning tips do not mention the concept card, users can also view the concept card very conveniently.

Reply:
As a note, I'm fine with the button, but I suggest you make it say "View Concept Card" (i.e. it should be an action).

Reply:
Can we atleast change the text to '...try reviewing the concept card **below**...'. Basically directing the users to that button. Then we don't need the link. My main worry is that there is no proper connection between the text and the button. And like Sean said, make the button text as 'View Concept Card'

Reply:
Done.